src/marketing/index.scss

```diff
@@ -11,4 +11,5 @@
 // marketing specific css modules
 @import './type/index.scss';
 @import './buttons/index.scss';
+@import './links/index.scss';
```

Comment:
Could it make more sense to just have a `links.scss` here, rather than adding both `links/index.scss` and `links/links.scss`? 👀

Reply:
I was wondering the same, but saw that all the other Primer components are following this structure and wanted to follow the convention, in case there is a good reason for it.